src/app/lib/utilities/isTenHoursAgo/index.js

```diff
@@ -1,3 +1,11 @@
+/**
+ * This function actually checks if the relative time is ten hours ago OR LESS.
+ * It will be true when 1 millisecond before 10 hours, on the exact millisecond of 10 hours.
+ * And when 1 millisecond passes after 10 hours it will return false.
+ * The tests take this into account.
+ * The users will be very unlikely to ever see '10 hours ago' written.
+ * The argument milliseconds can either be a number which is the epoch, or a string which is the ISO string
```

Comment:
This is not quite right - I think the param is explicitly named `milliseconds` to make sure that we send in the milliseconds-since-epoch value. 

This can be confirmed via the usage in src/app/legacy/components/Promo/timestamp.jsx above.

```
const isRelative = isTenHoursAgo(new Date(children).getTime());` 
```

`getTime()` returns the milliseconds since the epoch for the specified date & time of the children (however the Date constructor (new Date()) can accept either an epoch number or string in ISO format).

Let's either reword or remove this.

Reply:
oh yeah, this was the fix to make it right. I was still thinking about how something else was sometimes being sent in before our changes, and that is NOT what we want. Will change.